NuRadioMC/simulation/simulation.py

```diff
@@ -27,14 +32,29 @@
 from NuRadioReco.framework.parameters import channelParameters as chp
 from NuRadioReco.framework.parameters import electricFieldParameters as efp
 from NuRadioReco.framework.parameters import eventParameters as evp
+from NuRadioReco.framework.parameters import showerParameters as shp
 import datetime
 import logging
 from six import iteritems
 import yaml
 import os
 import collections
-# import confuse
-logger = logging.getLogger("sim")
+
+STATUS = 31
```

Comment:
What does this variable mean?

Reply:
I improved the output. We had a lot of into messages via logger.warning which confused people. So I added a new custom "status" message `logger.status("")` which is just a big higher than warning but below info.